plugins/action_update_review_status/src/databaseMethods.ts

```diff
@@ -9,13 +9,39 @@ const databaseMethods = (DBConnect: any) => ({
       review_assignment.level_number AS "levelNumber",
       review_assignment.id AS "reviewAssignmentId",
       review_status_history.status AS "reviewStatus"
-      FROM review JOIN review_assignment
-      ON review.review_assignment_id = review_assignment.id
-      JOIN review_status_history ON review.id = review_status_history.review_id
-      WHERE review_status_history.is_current = true
-      AND review_assignment.application_id = $1
-      AND stage_id = $2
-      AND review_assignment.level_number = $3
+      FROM review_assignment 
+        LEFT JOIN review ON review_assignment.id = review.review_assignment_id 
+        LEFT JOIN review_status_history ON review.id = review_status_history.review_id 
+          AND review_status_history.is_current = true
```

Comment:
Just wondering -- we currently have created Views for application_stage and status history (`application_stage_status_all` and `application_stage_status_latest`) which has been quite useful. Do you think we should have one for review histories too?

Reply:
Hmm perhaps. Do you think I could include as part of this PR?

Reply:
If it's not too much extra work. But can make a new issue if you'd prefer.

Reply:
Hmm I was checking out file `32_review_status_history` and saw that the latest `status` is already exposed in the `review` table:
https://github.com/openmsupply/application-manager-server/blob/9da7a3cb43932aac717af2267430b56387d19039/database/buildSchema/32_review_status_history.sql#L42-L52
So actually we don't need this view... I can just change this code to use review.status instead.

Reply:
Looks like "status" is visible on `review` table in GraphQL, but not in SQL, as it hasn't been added as a generated column.

Reply:
Yes, tried changing and realised that too. So will just leave this one as is.